### PR TITLE
Issue962 regression test

### LINKED ISSUE #962: Unit test in package AixLib.Fluid.Movers.PumpsPolynomialBased.Examples
**What is the problem?**

- Error when setting up unit tests for package AixLib.Fluid.Movers.PumpsPolynomialBased.Examples

**How do we want to solve it?**
- The package structure is not correct. For this reason the folder structure should be corrected.

### PR DESCRIPTION
**Pull request by DaJansenGit**
this closes #962

### CODE REVIEW

File: bin/02_CITests/SyntaxTests/StyleChecking.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def _CheckStyle(self):
 				self.Changedmodels = False
 			else:
 				for l in model_list:
-					print("Check package or model "+ l)
+					#print("Check package or model "+ l)
```
Comment (PMehrfeld) [inline bin/02_CITests/SyntaxTests/StyleChecking.py:99]: @DaJansenGit: Was this on purpose?

File: bin/02_CITests/SyntaxTests/StyleChecking.py [hunk 2]
```diff
@@ -147,7 +147,7 @@ def _StyleCheckLog_Check(self):
 			if len(line) == 0:
 				continue
 			else:
-				print("Error in model: \n \n"+line.lstrip())
+				#print("Error in model: \n \n"+line.lstrip())
```
Comment (PMehrfeld) [inline bin/02_CITests/SyntaxTests/StyleChecking.py:150]: same here

File: AixLib/Systems/HydraulicModules/BaseClasses/PumpInterface_PumpHeadControlled.mo [hunk 3]
```diff
@@ -1,113 +1,21 @@
...
 
   // Assumptions
-  parameter Boolean checkValve=false "= true to prevent reverse flow"
-    annotation (Dialog(tab="Assumptions"), Evaluate=true);
   parameter Modelica.SIunits.Volume V=0 "Volume inside the pump"
     annotation (Dialog(tab="Assumptions"), Evaluate=true);
 
   // Power and Efficiency
-  parameter Boolean calculatePower=false "calc. power consumption?"
+  parameter Boolean calculatePower=true "calc. power consumption?"
```
Comment (PMehrfeld) [inline AixLib/Systems/HydraulicModules/BaseClasses/PumpInterface_PumpHeadControlled.mo:18]: @alexanderAKU : Why setting here `calculatePower=true` but not in model `PumpInterface_PumpSpeedControlled`?

**Comment by PMehrfeld** [comment]
Generally I agree. However, I added some notes as in-line comments.  
Some more comments @alexanderAKU :

- Why have all the assertions been deleted in the pump models? They seemed to have a meaning. And assertions are (normally) always good ;)
- Besides setting a pump volume `V`, I also like the approach of setting a time constant, like in the IBPSA pump models. The only difference is to calculate a `V` internally depending on a (new) parameter `tau` and `m_flow_nominal`. See `AixLib.Fluid.Delays.DelayFirstOrder`.
- Keep the simulate and plot functions as clear and small as possible. You do not need to set ranges or colors. This is only done by getting the plot setup with the Dymola command. I think [this file](https://github.com/RWTH-EBC/AixLib/blob/development/AixLib/Resources/Scripts/Dymola/BoundaryConditions/InternalGains/Examples/HumansSenTDep.mos) shows a good example.

I set @alexanderAKU as assignee so that he can do changes (if desired) and merge afterwards.

File: bin/02_CITests/SyntaxTests/StyleChecking.py (hunk 1 above)
Comment (DaJansenGit) [reply bin/02_CITests/SyntaxTests/StyleChecking.py:99]: these checks were made by @SvenHinrichs. I think they are on purpose, but please correct me if not Sven.

File: bin/02_CITests/SyntaxTests/StyleChecking.py (hunk 2 above)
Comment (DaJansenGit) [reply bin/02_CITests/SyntaxTests/StyleChecking.py:150]: these checks were made by @SvenHinrichs. I think they are on purpose, but please correct me if not Sven.

**Comment by PMehrfeld** [comment]
CI failed, last line in dymola-log.txt is: `savelog("AixLib.Fluid.Movers.PumpsPolynomialBased.Examples.PumpHeadControlledDpV.translation.log");`

@alexanderAKU : If you can't find a solution, we might have a more precise look into this together :)

File: AixLib/Systems/HydraulicModules/BaseClasses/PumpInterface_PumpHeadControlled.mo (hunk 3 above)
Comment (alexanderAKU) [reply AixLib/Systems/HydraulicModules/BaseClasses/PumpInterface_PumpHeadControlled.mo:18]: should be true in both cases. I will fix this.

**Comment by DaJansenGit** [comment]
> CI failed, last line in dymola-log.txt is: `savelog("AixLib.Fluid.Movers.PumpsPolynomialBased.Examples.PumpHeadControlledDpV.translation.log");`
> 
> @alexanderAKU : If you can't find a solution, we might have a more precise look into this together :)

Thats quite interesting, because these reference results have been newly created.

**Comment by DaJansenGit** [comment]
@SvenHinrichs anything new here? :)

**Review by alexanderAKU — APPROVED** [review]

**Review by DaJansenGit — COMMENTED** [review]
content was already reviewed. New reference results are implemented.

**Pull request merged**